lib/metaphysics.js

```diff
@@ -1,10 +1,11 @@
 /* @flow */
 'use strict'
 
-import { NativeModules, Promise } from 'react-native'
+import { NativeModules } from 'react-native'
```

Comment:
this use of `Promise`. is consistent with the native ES6 one, and so there's no need to import RNs one